src/PostgREST/DbStructure.hs

```diff
@@ -710,13 +710,13 @@ allComputedRels =
     ),
     computed_rels as (
       select
-        p.pronamespace::regnamespace::text as schema,
-        p.proname::text                    as name,
-        arg_schema.nspname::text           as rel_table_schema,
-        arg_name.typname::text             as rel_table_name,
-        ret_schema.nspname::text           as rel_ftable_schema,
-        ret_name.typname::text             as rel_ftable_name,
-        p.prorows = 1                      as single_row
+        (parse_ident(p.pronamespace::regnamespace::text))[1] as schema,
```

Comment:
I could not find another way of removing the double quote without using a join to `pg_namespace`.

Reply:
That one is pretty good actually. TIL about `parse_ident`